changelog/_unreleased/2025-03-07-make-offcanvas-navigation-accessible.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Make off-canvas navigation accessible
+issue: NEXT-40780
+---
+# Storefront
+* Changed the `offcanvas-menu.plugin.js` to fix several issues and make it accessible via keyboard:
+  * Removed the animation. The sub-category overlay will now simply be shown, which fixes additional issues like unnecessary scrollbars.
+  * Removed some unnecessary wrapper elements to simply the DOM structure.
+  * Removed a lot of unnecessary methods according to the simplified behaviour.
```

Comment:
The methods are private, but maybe we can still list them in the changelog.

Reply:
Added more details 👍